packages/flutter_tools/test/integration.shard/variable_expansion_windows_test.dart

```diff
@@ -17,5 +17,5 @@ void main() {
       '"^(?!Golden).+"',
     ]);
     expect(result.stdout, contains('args: ["(?!Golden).+"]'));
-  }, skip: !platform.isWindows);
+  }, skip: 'Reverted in https://github.com/flutter/flutter/pull/86000');
```

Comment:
Huh. I didn't know this could be a string. Does this just act like `true`, then?

Reply:
Yup, but I think it will display in the test output